.github/workflows/test-suite.yml

```diff
@@ -78,3 +78,31 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           config: baseUrl=http://127.0.0.1:8000
+
+  Psalm:
+    name: Psalm
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick
+          coverage: xdebug
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache/files
+          key: dependencies-laravel-8.x-php-8.0-composer-${{ hashFiles('composer.json') }}
```

Comment:
Do I need to also cache the dependencies if it is already being cached in the Code-Climate-tests pipeline?

Reply:
I believe you do, seeing as the code climate and psalm pipelines run on different virtual machines -- no harm either way